assets/app/views/browse/image.html

```diff
@@ -57,96 +57,106 @@
+                          {{tag.status.items[0].dockerImageReference}}
                         </div>
-                      </td>
-                    </tr>
-                    <tr ng-repeat="item in tag.status.items" ng-if="tagShowOlder[tag.name] && !$first && item.image">
-                      <td>&nbsp;</td>
-                      <td>&nbsp;</td>
-                      <td>
-                        <short-id id="{{item.image | imageName}}"></short-id>
-                      </td>
-                      <td>
-                        <relative-timestamp timestamp="item.created"></relative-timestamp>
-                      </td>
-                      <td>
-                        <div ng-if="item.dockerImageReference" style="max-width: 400px;" class="truncate"
-                               ng-attr-title="{{item.dockerImageReference}}">
-                           {{item.dockerImageReference}}
+                        <div class="pull-spec truncate hidden-xs"
+                         ng-attr-title="{{tag.status.items[0].dockerImageReference}}">
+                          {{tag.status.items[0].dockerImageReference}}
+                        </div>                      
+                      </div>
+                    </td>
+                  </tr>
+                </tbody>
+                <tbody ng-repeat="item in tag.status.items" ng-if="tagShowOlder[tag.name] && !$first && item.image">
+                  <tr>
+                    <td data-title="Tag"><span class="hidden-xs">&nbsp;</span><span class="visible-xs">{{tag.name}}</span></td>
+                    <td class="hidden-xs">&nbsp;</td>
+                    <td data-title="Older Image">
+                      <short-id id="{{item.image | imageName}}"></short-id>
+                    </td>
+                    <td data-title="Created">
+                      <relative-timestamp timestamp="item.created"></relative-timestamp>
+                    </td>
+                    <td data-title="Pull Spec">
+                      <div ng-if="item.dockerImageReference">
+                        <div ng-attr-title="{{item.dockerImageReference}}" class="word-break visible-xs-block">
+                          {{item.dockerImageReference}}
                         </div>
-                      </td>
-                    </tr>
-                    <tr ng-repeat-end style="display: none;"></tr>
-                  </tbody>
-                </table>
-              </div>
+                        <div class="pull-spec truncate hidden-xs"
+                         ng-attr-title="{{item.dockerImageReference}}">
+                          {{item.dockerImageReference}}
+                        </div>                      
+                      </div>                      
+                    </td>
+                  </tr>
+                </tbody>
+                <tbody ng-repeat-end style="display: none;"></tbody>
```

Comment:
multiple sibling elements (tbody) that need to be repeated without the parent being repeated (table),  the third tbody is to end the outer ng-repeat

```
<tbody ng-repeat-start="the tag"></tbody>
<tbody ng-repeat="previous images for this tag"></tbody>
<tbody ng-repeat-end /* ends the outer repeat */ ></tbody>
```

I was skeptical whether having both an ng-repeat-end and an ng-repeat on the second tbody would work correctly, hence the third one.